### ISSUE TITLE
Missing zero_grad in text classifier tutorial

### ISSUE THREAD

**Issue by wulu473** [pytorch/opacus#194]
The `building_text_classifier.ipynb` tutorial never calls `optimizer.zero_grad()`. The accumulated gradients should be zeroed after a step is taken.

**Comment by JohnlNguyen** [comment]
zero_grad is not needed for opacus. Please see https://github.com/pytorch/opacus/pull/96

for more details
